packages/iris-grid/src/GotoRow.tsx

```diff
@@ -115,9 +115,9 @@ const GotoRow = forwardRef<GotoRowElement, GotoRowProps>(
     };
 
     const handleGotoValueKeySubmit = (
-      e: KeyboardEvent<HTMLInputElement>
+      e?: KeyboardEvent<HTMLInputElement>
```

Comment:
This doesn't seem right. I think this should be guaranteed to get the event and instead you should go down through where this is passed and make those non-optional params as well. Looks like it goes through `DateTimeInput` and `MaskedInput`